src/build.d

```diff
@@ -533,6 +534,55 @@ alias zip = memoize!(function()
     return new DependencyRef(dep);
 });
 
+string d2html(string sourceFile)
+{
+    auto ext = sourceFile.extension();
+    assert(ext == ".d" || ext == ".di", sourceFile);
+    const htmlFilePrefix = (sourceFile.baseName == "package.d") ?
+        sourceFile[0 .. $ - "package.d".length - 1] :
+        sourceFile[0 .. $ - ext.length];
+    return htmlFilePrefix ~ ".html";
+}
+
+alias html = memoize!(function()
+{
+    Dependency htmlDep;
+    htmlDep.name = "html";
+    htmlDep.description = "Generate html docs, requires DMD and STDDOC to be set";
+    auto docSources = .sources.root ~ .sources.lexer ~ .sources.dmd ~ env["D"].buildPath("frontend.d");
+    if (env.get("DMD", null).length == 0) {
+        htmlDep.commandFunction = () {
+            writefln("ERROR: cannot build '%s' unless DMD is set", htmlDep.name);
+        };
+    } else {
+        htmlDep.msg = "HTML " ~ docSources.join(" ");
+        htmlDep.deps = new DependencyRef[docSources.length];
+        const stddocs = env.get("STDDOC", "").split();
+        foreach(i; 0 .. docSources.length) {
+            Dependency docDep;
+            docDep.sources = docSources[i..i+1];
+            docDep.target = env["DOC_OUTPUT_DIR"].buildPath(d2html(docSources[i])[srcDir.length + 1..$]
+                .replace(dirSeparator, "_"));
+            docDep.command = [
+                env["DMD"],
+                "-o-",
+                env["MODEL_FLAG"],
+                "-c",
+                "-Dd" ~ env["DOCSRC"],
+                "-J"~env["G"], "-J../res",
+                "-I" ~ env["D"],
+                srcDir.buildPath("project.ddoc")
+            ] ~ stddocs ~ [
+                "-Df" ~ docDep.target,
+                docSources[i]
+            ] ~ flags["DFLAGS"];
+            docDep.msg = docDep.command.join(" ");
```

Comment:
```suggestion
            docDep.msg = docDep.target
```

Less noisy, `--dry-run` already exists to dump the invoked command

Reply:
we want to see the commands in the CI builders though.`--dry-run` is for developers